jujugui/static/gui/src/test/test_utils.js

```diff
@@ -1163,4 +1163,69 @@ describe('utilities', function() {
     });
   });
 
+  describe('linkify', function() {
```

Comment:
Added a slew of tests to check various URLs, from the good to the bad to the ugly.